examples/src/main/java/org/apache/spark/examples/JavaSparkPi.java

```diff
@@ -34,6 +34,7 @@ public static void main(String[] args) throws Exception {
     SparkSession spark = SparkSession
       .builder()
       .appName("JavaSparkPi")
+      .config("spark.master", "local")
```

Comment:
This example is supposed to run with, for example, `./bin/run-example JavaSparkPi 10`. I think it's fine without the master.